### PR TITLE
Copter: Adding a class name

### PR DESCRIPTION
**Pull request by muramura**
In other methods, the class name is appended.
I match the format.

### CODE REVIEW

**Review by rmackay9 — CHANGES_REQUESTED** [review]
I'm ok with this change but if we're going to change one, let's change all calls to micros() to specify the class.  So a single commit for Copter and Rover would be acceptable.  I'm less sure if the Blimp and Plane maintainers will accept the change.

File: ArduCopter/failsafe.cpp [hunk 1]
```diff
@@ -18,7 +18,7 @@ static bool in_failsafe;
 void Copter::failsafe_enable()
 {
     failsafe_enabled = true;
-    failsafe_last_timestamp = micros();
```
Comment (peterbarker) [inline ArduCopter/failsafe.cpp:21]: I think I recall losing the namespace actually makes this faster.

Comment (peterbarker) [reply ArduCopter/failsafe.cpp:21]: Confirmed, this is faster and smaller.

**Comment by tridge** [comment]
we deliberately have it in vehicles as faster and smaller

**Comment by rmackay9** [comment]
@muramura, feel free to raise a PR to instead remove the AP_HAL:: part from other calls in the Copter code.

**Pull request closed without merging**